README.md

```diff
@@ -28,6 +28,143 @@ Let's simulate a simple arm whose elbow is actuated by a muscle, using
 the C++ interface (You can find a Python version of this example at
 `Bindings/Python/examples/build_simple_arm_model.py`):
 
+<details>
+<summary>Python</summary>
+ 
+```py
+import opensim as osim 
+
+# Define global model where the arm lives.
+arm = osim.Model()
```

Comment:
To be consistent with the C++ code, this line should be `model = osim.Model("bicep_curl")`.